aws/resource_aws_pinpoint_app_test.go

```diff
@@ -0,0 +1,249 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/pinpoint"
```

Comment:
Nitpick: We prefer to run [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports) to automatically format imports, which would move this entry to the bottom imports section. Many editors support automatically running it on save.

Reply:
Fixed in https://github.com/terraform-providers/terraform-provider-aws/pull/5956/commits/293a621cbff28eb95487ebda815f724fd57381a3 and https://github.com/terraform-providers/terraform-provider-aws/pull/5956/commits/ea1e963c22edd9cc3720899eecedc4bd13f36c6d